radish/compat.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+"""
+    This module provide Oython 2 / Python 3 compatability
+"""
+
+def unicode_(text):  # pragma: no cover
```

Comment:
Do you think it would make sense to just use `u` as a function name? 
This way we wouldn't pollute code and we could use it in different other places where the `unicode_(...)` might be too verbose?